tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "moduleResolution": "node",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
-    "target": "es2018",
+    "target": "es2020",
```

Comment:
Are we pretty confident that everyone who visits Dockstore will have a browser that supports `es2020`?  Related: https://kangax.github.io/compat-table/es2016plus/

Reply:
> Are we pretty confident that everyone who visits Dockstore will have a browser that supports `es2020`? Related: https://kangax.github.io/compat-table/es2016plus/

Hmm, nice catch, I hadn't even noticed that.

Looking into it, that is the Angular 14 default (their migration tools updated it without me noticing). Looking at your link, all desktop browsers support it, except Safari doesn't support Function.prototype.toString(), which is in es2019. It's not a feature we rely on (gets the source code of a function).

The only one that scares me is Samsung mobile browsers not support optional chaining... So I lean towards dropping to 2019. Thoughts by anybody else?

Reply:
> The only one that scares me is Samsung mobile browsers not support optional chaining... So I lean towards dropping to 2019. Thoughts by anybody else?

FWIW, in the last 30 days, according to Google Analytics, mobile users are 4.18% of users. 
Alternatively by browser, "Samsung Internet" is .09% of all possible Browser & OS combinations. 
That said, some folks may be blocking GA. And of course, this could be a chicken and the egg problem due to issues like https://github.com/dockstore/dockstore/issues/5144

Reply:
Looking more closely at Analytics, we're only getting 14 and 18 of Samsung Internet -- in the table, version 14 does support optional chaining, although not with the spread operator after.

I'm inclined to leave it at es2020. Last chance for anybody to push back.